### PR TITLE
Feat: Laminas coding style 2.1

### PR DESCRIPTION
**Pull request by geerteltink**
This PR adds Laminas coding style 2.0.

|    Q          |   A
|-------------- | ------
| New Feature   | yes/no

### CODE REVIEW

File: phpcs.xml [hunk 1]
```diff
@@ -1,8 +1,36 @@
 <?xml version="1.0"?>
 <ruleset name="Laminas coding standard">
-    <rule ref="./vendor/laminas/laminas-coding-standard/ruleset.xml"/>
-
     <!-- Paths to check -->
     <file>src</file>
     <file>test</file>
+
+    <rule ref="LaminasCodingStandard">
+        <exclude name="WebimpressCodingStandard.Files.DeclareStrictTypes.BelowComment"/>
```
Comment (geerteltink) [inline phpcs.xml]: This rule must be removed from the coding standard. 

see laminas/laminas-coding-standard#35

Comment (Ocramius) [reply phpcs.xml]: Should we wait for an upstream release, or is this to be merged first?

Comment (geerteltink) [reply phpcs.xml]: Yes, that's why I just released a new coding standard version.

Comment (geerteltink) [reply phpcs.xml]: Done.

**Pull request merged**